data/language/en-GB.txt

```diff
@@ -3661,6 +3661,15 @@ STR_6406    :GOG RollerCoaster Tycoon 2 Installer
 STR_6407    :This may take a few minutes.
 STR_6408    :Please install “innoextract” to extract GOG Installer, then restart OpenRCT2.
 STR_6409    :The selected file is not the offline GOG Installer for RollerCoaster Tycoon 2. You may have downloaded the GOG Galaxy downloader stub or selected the wrong file.
+STR_6410    :{WINDOW_COLOUR_2}Your Progress:
+STR_6411    :{BLACK}-
+STR_6412    :{BLACK}Guests: {COMMA32} of {COMMA16}
+STR_6413    :{BLACK}Rating: {COMMA16} of {INT32}
```

Comment:
Different format types used here, should both be `COMMA16`

Reply:
The gParkRating variable is a uint16_t, while the target rating are the hard-coded numbers 600 and 700 which should be represented as int32_t. Or should I cast them to uint16_t?

Reply:
They should both be comma separated. As for size, I'd make them both 32-bit and cast as necessary.

Reply:
OK, I'll do that. :)